projects/igniteui-angular/schematics/ng-add/index.spec.ts

```diff
@@ -99,4 +129,19 @@ describe('schematics', () => {
     expect(Object.keys(pkgJsonData.devDependencies).filter(k => k.includes('igniteui-cli')).length).toBeGreaterThan(0);
     expect(Object.keys(pkgJsonData.dependencies).filter(k => k.includes('igniteui-cli')).length).toBe(0);
   });
+
+  it('should properly add polyfills', () => {
+    expect(tree.readContent('src/polyfills.ts')).toBe(polyfills);
+    runner.runSchematic('ng-add', { polyfills: true }, tree);
+    expect(tree.readContent('src/polyfills.ts')).not.toBe(polyfills);
```

Comment:
While technically true the file shouldn't remain the same, it's not really an exact check the schematic did the expected job :)